backend/python/app/utilities/test_csv.py

```diff
@@ -0,0 +1,161 @@
+"""
```

Comment:
Can we move this test file to be under `backend/python/tests` instead?

Reply:
Moved to functional folder

Reply:
Hmm, I think `unit` might be more suitable